packages/block-library/src/button/block.json

```diff
@@ -89,7 +89,30 @@
 				"radius": true
```

Comment:
I know this probably isn't what anyone wants to hear, but should we consider splitting out the implementation details from the concrete usage on the Button block?

That way we can incrementally introduce the capability to new blocks one by one and check and merge PRs as we go.